UPDATING.md

```diff
@@ -70,6 +70,11 @@ https://developers.google.com/style/inclusive-documentation
 
 -->
 
+### Deprecated PodDefaults and in airflow.kubernetes.pod_generator
+
+We have moved PodDefaults and add_xcom_sidecar to `airflow.providers.cncf.kubernetes.utils.xcom_sidecar`.
+This change will allow us to modify the KubernetesPodOperator XCom functionality without requiring airflow upgrades.
```

Comment:
We should add the original location too -- that it is moved from XYZ to ABC

Reply:
@kaxil fixed